language-configuration.json

```diff
@@ -21,6 +21,8 @@
     ["(", ")"],
     ["\"", "\""],
     ["'", "'"],
-    ["`", "`"]
+    ["`", "`"],
+    ["_", "_"],
+    ["**", "**"]
```

Comment:
Double asterisks won't work, change them into single asterisks. If you want to surround selected text with double asterisks, simply press the asterisk key twice. 😉